modules/openapi-generator/src/main/resources/dart2/pubspec.mustache

```diff
@@ -4,6 +4,6 @@ description: {{pubDescription}}
 environment:
   sdk: '>=2.0.0 <3.0.0'
 dependencies:
-  http: '>=0.11.1 <0.13.0'
+  http: '>=0.12.0 <0.13.0'
```

Comment:
@nickmeinhold To make the change backward compatible, what about revert the changes you've made and only make the following change?

`  http: '>={{^browserClient}}0.12.0{{/browserClient}}{{#browserClient}}0.11.1{{/browserClient}} <0.13.0'`

Reply:
Yeah absolutely, if you think that's the way to go.  My thinking was that because 0.12.0 was where the http package started using conditional imports, it was important to bump the dependency, as the change to [api_cient](modules/openapi-generator/src/main/resources/dart2/api_client.mustache) assumes conditional imports.  Does that make sense?

Reply:
Right. I just want this change to be backward compatible so that it can be merged into master right away. Otherwise, the change should be targeting 4.2.0 branch, which will be released 2 months later.

Reply:
OK cool, I'll make the change.

Reply:
Oh shoot, I think I misunderstood - you're suggesting I revert all the other changes so the whole PR is just a single change of that one line, is that right?